src/appmixer/system/core/OnAnyFlowComponentError/OnAnyFlowComponentError.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+module.exports = {
+
+    async start(context) {
+
+        // Get current WEBHOOK_FLOW_COMPONENT_ERROR from system config
+        const config = await context.callAppmixer({
+            endPoint: '/config',
+            method: 'GET'
+        });
+        const urls = config.find(c => c.key === 'WEBHOOK_FLOW_COMPONENT_ERROR')?.value?.split(',') || [];
+        // Use enqueueOnly to automatically enqueue the webhook in the `receive` method.
+        const newUrl = context.getWebhookUrl({ enqueueOnly: true });
+        const updatedUrls = [...urls, newUrl].join(',');
+
+        await context.log({ step: 'Update config', newUrls: updatedUrls, oldUrls: urls });
+        // Update WEBHOOK_FLOW_COMPONENT_ERROR in system config
+        await context.callAppmixer({
```

Comment:
There is a slight chance of messing things up. If two of these components are started simultaneously, one URL could get lost. That should not happen, since this is meant for admins only. And it does not make much sense to have more than 1 in the system. But using `locks` would make it safer. 
But even with a lock, there is a small chance, that the admin changes this in the Backoffice at the same time ...

Reply:
I can imagine a case where admin is checking for different error types - for example per connector. Thus using this component multiple times.

I added a lock named `OnAnyFlowComponentError` for both stop and start functions. Could we use this new lock created by the component also in the Backoffice?